src/components/footer/footer.hbs

```diff
@@ -6,10 +6,10 @@
                     <a href="#" class="footer__main-link" data-modal-show="construction-modal">Site Map</a>
                 </li>
                 <li class="footer__main-link-item">
-                    <a href="#" class="footer__main-link" data-modal-show="construction-modal">About</a>
+                    <a href="{{rootPath}}/index.html" class="footer__main-link">About</a>
```

Comment:
The `rootPath` will be undefined. You need to pass it in through `scripts.js`. Use the same `globalVariables` object